### ISSUE TITLE
Question: Is there a delay to return the new orderId?

### ISSUE THREAD

**Issue by diogojuliano89** [anjlab/android-inapp-billing-v3#294]
In the Google Play Console, when the subscription is renewed, create a new orderId with "..0" or "..1", but when I use bp.getSubscriptionTransactionDetails (), the orderId does not change. Is there a delay to return the new orderId?

**Comment by AllanWang** [comment]
It may be because the transactions are cached. Have you tried calling `loadOwnedPurchasesFromGoogle`?

**Comment by diogojuliano89** [comment]
Yes, on onBillingInitialized () I call loadOwnedPurchasesFromGoogle. I'm trying with a test account. In production, will OrderId update?

**Comment by smaidah** [comment]
I am also facing this issue. any update on this ?
